src/index.js

```diff
@@ -156,25 +157,26 @@ export const validateModulesOption = (modulesOpt = "commonjs") => {
   return modulesOpt;
 };
 
-export const validateWhitelistOption = (whitelistOpt = []) => {
-  if (!Array.isArray(whitelistOpt)) {
-    throw new Error(`The 'whitelist' option must be an Array<string> of plugins
-      {
-        "presets": [
-          ["env", {
-            "targets": {
-              "chrome": 50
-            },
-            "whitelist": ["transform-es2015-arrow-functions"]
-          }]
-        ]
-      }
-      was passed "${whitelistOpt}" instead
-    `);
+function validatePluginsOption(opts = [], type) {
+  if (!Array.isArray(opts)) {
+    throw new Error(`The '${type}' option must be an Array<string> of plugins`);
   }
 
-  return whitelistOpt;
-};
+  for (let opt of opts) {
+    if ([
+      ...Object.keys(pluginFeatures),
+      ...Object.values(MODULE_TRANSFORMATIONS)
+    ].indexOf(opt) === -1) {
```

Comment:
This is in the transform so we can't use new features unless we want to use transform-runtime again or we drop old node